[ENG-29890] feat: accounting endpoint (#894)
aziontech/docs: src/content/docs/en/pages/devtools/api-graphql/features/accounting-fields.mdx

---
title: Accounting GraphQL API Fields
description: See descriptions for all fields from each Accounting GraphQL API dataset.
meta_tags: graphql, api, azion, fields, accounting
namespace: documentation_graphql_accounting_fields
permalink: /documentation/devtools/graphql-api/features/gql-accounting-fields/
menu_namespace: graphqlMenu
---

Each [GraphQL API dataset](/en/documentation/devtools/graphql-api/features/#datasets) has specific fields that can be used when querying for data from that dataset.

See each available field and their descriptions below.

---

## accountingDetail (Accounting)

| Field | Description |
| ----- | ----------- |

> | accounted | Total amount of use of products per region accounted for the bill detail of the client. Example: `1767504904` |

Reviewer: I don't know if it's a good suggestion, i'm actually an engineer. 🧑‍🏭  😄 
```suggestion
| accounted | Total amount of product usage per region accounted in the bill detail of the client. Example: `1767504904` |
```

| clientId | Client unique identifier on Azion. Example: `8437r` |
| metricSlug | Identifier of the metric being account. See the complete list of product metrics in the [Terms of Service](/en/documentation/agreements/tos/#3-product-use-calculation-metrics-and-methods) document. Example: `load_balancer_data_transferred` |
| periodFrom | Start date of the accounted data in year-month-day format. Example: `2022-06-01` |
| periodTo | End date of the accounted data in year-month-day format. Example: `2022-06-30` |
| productSlug | Identifier of the product being accounted. Example: `edge_functions` |

> | regionName | Region that the metric was accounted to. Find more information on the [Pricing](/en/documentation/products/pricing/) page. Can be: `United States`, `Canada`, `Europe`, `Brazil`, or `Latam` |

Author: ```suggestion
| regionName | Region that the metric was accounted to. Find more information on the [Pricing](/en/documentation/products/pricing/) page. Can be: `United States`, `Canada`, `Europe`, `Brazil`, `Latam`, `All Other Regions`, or `No Region` |
```

> | regionName | Region that the metric was accounted to. Find more information on the [Pricing](/en/documentation/products/pricing/) page. Can be: `United States`, `Canada`, `Europe`, `Brazil`, or `Latam` |

Reviewer 2: Other Regions não aplica aqui?
